### ISSUE TITLE
Question about photometric loss.

### ISSUE THREAD

**Issue by SepJourney** [LinShan-Bin/OccNeRF#8]
Thanks for sharing this nice work!
Sorry, I can't run the code directly in my environment. One question I have is what the input to photometric loss looks like.

loss = L1_loss(pred, target), are the pred and target 3-channels RGB images? 
Whether the value ranges from 0 to 255 (original image) or normalized value (normalized by mean/std value)? I don't see a place in the code for image normalization.

**Comment by weiyithu** [comment]
They are 3-channels RGB images ranged from 0 to 1.

**Comment by SepJourney** [comment]
> They are 3-channels RGB images ranged from 0 to 1.

Got it, thanks for the replay.
